modules/rs_grading.py

```diff
@@ -2,15 +2,12 @@
 import logging
 from math import ceil
 from psycopg2 import IntegrityError
+
+from gluon import current
 from outcome_request import OutcomeRequest
 
-# When testing, the ``settings`` object isn't defined. Import it in this case.
-try:
-    logger
-except:
-    execfile('applications/runestone/models/0.py', globals())
-logger = logging.getLogger(settings.logger)
-logger.setLevel(settings.log_level)
+logger = logging.getLogger(current.settings.logger)
+logger.setLevel(current.settings.log_level)
```

Comment:
We should make a todo  that functions like _scorable_mchoice_answers no longer need to pass db as a parameter.

Reply:
Good point. Do you want me to submit an issue? Or would you like to?